veros/core/density/gsw.py

```diff
@@ -545,3 +545,489 @@ def gsw_dHdS(vs, sa_in, ct_in, p):
                                                                              (t4 + t6 - t242)
                                                                              - 2.0 * t237 * t200 / t247 * (t4 + t6 + t242 + t123)) / t207
     return t260
+
+"""
+==========================================================================
+  Potential density & temperature, enthropy, Gibbs free energy and its
+  derivatives from Absolute Salinity, in-situ Temperature, Sea Pressure
+  using the Thermodynamic Equation Of Seawater - 2010 (TEOS-10) package
+  (see details in IOC, 2010; http://www.teos-10.org)
+==========================================================================
+"""
+@veros_method
+def get_gsw_teos10_constants(vs, key):
+
+    constants = dict(\
+
+    db2pa = 1.0e4,\
```

Comment:
Line continuation characters (`\`) are not needed inside bracketed expressions, and should be removed.